shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java

```diff
@@ -77,7 +77,9 @@ public void onMethodCall(MethodCall call, Result result) {
         }
     }
 
-    private static int inputTypeFromTextInputType(String inputType, boolean obscureText) {
+    private static int inputTypeFromTextInputType(String inputType,
+                                                  boolean obscureText,
+                                                  boolean noAutoCorrect) {
```

Comment:
FWIW one of the lessons we learnt on the Web was that it's less confusing on the long run for the argument or property to be phrased in the positive than for "true" to be the default. We have several boolean arguments that default to true in the framework. (For example, we assiduously avoid having "hidden" and "disabled" arguments or properties.)